docs/discounts/discounts_api.md

```diff
@@ -0,0 +1,128 @@
+| `Ibexa\Discounts\Value\DiscountCondition\MinimumPurchaseAmount` | Cart | `minimum_purchase_amount` | Checks if purchase amount in the cart exceeds the specified minimum | `minimum_purchase_amount` |
+| `Ibexa\DiscountsCodes\Value\DiscountCondition\IsValidDiscountCode` | Cart | `is_valid_discount_code` | Checks if the correct discount code has been provided and how many times it was used by the customer | `discount_code`, `usage_count` |
+
+When multiple conditions are specified, all of them must be met.
+
+### Priority
+
+You can set discount priority as a number between 1 and 10 to indicate which discount should have [higher priority](discounts_guide.md#discounts-priority) when choosing the one to apply.
+
+### Start and end date
+
+Discounts can be permanent, or valid only in a specified time frame.
+
+Every discount has a start date, which defaults to the date when the discount was created.
+The end date can be set to `null` to make the discount permanent.
+
+### Status
+
+You can disable a discount anytime to stop it from being active, even if the conditions enforced by start and end date are met.
+
+Only disabled discounts can be deleted.
+
+### Discount translations
+
+The discount has four properties that can be translated:
+
+| Property | Usage |
+|---|---|
+| Name | Internal information for store managers |
+| Description | Internal information for store managers |
+| Promotion label | Information displayed to customers |
+| Promotion description | Information displayed to customers |
+
+Use the [DiscountTranslationStruct](/api/php_api/php_api_reference/classes/Ibexa-Contracts-Discounts-Value-Struct-DiscountTranslationStruct.html) to provide translations for discounts.
+
+### Discount codes
+
+To activate a cart discount only after a proper discount code is provided, you need to:
+
+1. Create a discount code using the [DiscountCodeServiceInterface::createDiscountCode()](/api/php_api/php_api_reference/classes/Ibexa-Contracts-DiscountsCodes-DiscountCodeServiceInterface.html#method_createDiscountCode) method
+1. Attach it to a discount by using the `IsValidDiscountCode` condition
+
+Set the [`usedLimit`](/api/php_api/php_api_reference/classes/Ibexa-Contracts-DiscountsCodes-Value-Struct-DiscountCodeCreateStruct.html#method___construct) property to the number of times a single customer can use this code, or to `null` to make the usage unlimited.
+
+The [`DiscountCodeServiceInterface::registerUsage()`](/api/php_api/php_api_reference/classes/Ibexa-Contracts-DiscountsCodes-DiscountCodeServiceInterface.html#method_registerUsage) method is used to track the number of times a discount code has been used.
+
+### Example API usage
+
+The example below contains a Command creating a cart discount. The discount:
+
+- has the highest possible priority value
+- deducts 10 EUR from the base price of the product
```

Comment:
```suggestion
- [rule](#rules) a deduction of 10 EUR from the base price of the product
```

Reply:
I've changed that to
```suggestion
- [rule](#rules) deducts 10 EUR from the base price of the product
```
It fits better in my head, but you're right that linking to `rule` section is good